DXRModules/DeusEx/Classes/DXRReduceItems.uc

```diff
@@ -377,7 +377,8 @@ simulated function SetMaxAmmo(class<Ammo> type, int percent)
         f = percent;
         f *= _GetItemMult(_max_ammo, a.class);
         a.MaxAmmo = float(a.default.MaxAmmo) * f / 100.0 * 0.8;
-        a.MaxAmmo = Clamp(a.MaxAmmo, 1, a.default.MaxAmmo*10);
+
+        a.maxAmmo = Clamp(a.MaxAmmo, class'DXRInfo'.static.Round(a.default.MaxAmmo/18.75), a.default.MaxAmmo*10);
```

Comment:
this /18.75 looks oddly specific and maybe not as clear as a multiplier like a percentage, maybe this would be better?
```suggestion
        a.maxAmmo = Clamp(a.MaxAmmo, Round(a.default.MaxAmmo * 0.05333), a.default.MaxAmmo*10); // 5.333% of default, for pistol this means 150=> 8, assault rifle 300 => 16
```